torchreid/optim/optimizer.py

```diff
@@ -7,6 +7,12 @@
 from .sam import SAM
 from .radam import RAdam
 
+NNCF_ENABLED = True
+try:
+    from nncf.torch.layer_utils import CompressionParameter
+except ImportError:
+    NNCF_ENABLED = False
```

Comment:
what if the nncf is installed but we do not want to train the model with nncf?

Reply:
Please do a function that returns necessary imported modules and variables
https://github.com/openvinotoolkit/deep-object-reid/blob/ote/torchreid/integration/nncf/compression.py

Reply:
Done